MantisPhpClient.php

```diff
@@ -295,14 +307,14 @@ public function getEnumViewState() {
      * Gets Project Users
      * @param int $p_project_id <p>Project id must be supplied</p>
      * @param int $p_access_id <p>Access id</p>
-     * @return Array <p> Result array</p>
+     * @return array <p> Result array</p>
      */
     public function getProjectUsers( $p_project_id, $p_access_id ) {
         return $this->soap_client->mc_project_get_users( $this->username, $this->password, $p_project_id, $p_access_id );
     }
 
     /**
-     * Get the list of accessible projects.
+     * @return array The list of accessible projects.
```

Comment:
I generally prefer having a line the describes the function that is separate from params and return value documentation.  I think this is better than just a return tag.  I prefer the function description to focus on the logic and return tag to focus on the value type returned, array format, possible values, can be null, etc.  Please apply to all methods that don't have a function description.